chat-plugins/trivia.js

```diff
@@ -1368,12 +1359,649 @@ module.exports = {
+			return this.sendReply(replyMsg);
+		} else {
+			return this.errorReply("Unable to vote at this time.");
+		}
+	},
+
+	pass: function (target, room, user) {
+		if (!room.game || room.game.gameid !== 'weakestlink') return this.errorReply("There is no game of the Weakest Link in progress.");
+		room.game.guess("", user, this.can("mute", null, room));
+	},
+
+	decide: function (target, room, user) {
+		if (!room.game || room.game.gameid !== 'weakestlink') return this.errorReply("There is no game of the Weakest Link in progress.");
+		room.game.decide(target, user);
+	},
+	checkplayers: function (target, room, user) {
+		if (!room.game || room.game.gameid !== 'weakestlink') return this.errorReply("There is no game of the Weakest Link in progress.");
+		if (!this.can('mute', null, room)) return;
+		if (!this.runBroadcast()) return;
+		return this.sendReplyBox(`<b>Remaining Players (${room.game.getRemainingPlayerCount()})</b>: ${Object.values(room.game.players).map(p => p.name).join(", ")}`);
+	},
+
+	help: function (target, room, user) {
+		let games = [
+			"<strong>Game commands</strong>:",
+			"- /wl new - Begin signups for a new Weakest Link game. Requires: % @ # & ~",
+			"- /wl join - Join a Weakest Link game during signups.",
+			"- /wl leave - Leave the game at any point.",
+			"- /wl start - Begin the game of the Weakest Link. Requires: % @ # & ~",
+			"- /guess [answer] - Guess your answer during your turn.",
+			"- /wl decide [user] - Decide which player to eliminate if a tie occurs during vote",
+			"- /wl pass - Skip the current question: Requires: % @ # & ~ or current player",
+			"- /wl endvote - End the voting phase. Requires: % @ # & ~",
+			"- /wl removeplayer [player] - Remove the given player from the game. Requires: % @ # & ~",
+		].join("<br />");
+		let questions = [
+			"<strong>Question modifying commands</strong>:",
+			"- /wl submit [question] | [answer1], [answer2] ... [answern] - Add a question to the Weakest Link submission database for staff to review.",
+			"- /wl review - View the list of submitted Weakest Link  questions. Requires: @ # & ~",
+			"- /wl accept [index1], [index2], ... [indexn] OR all - Add questions from the submission database to the Weakest Link question database using their index numbers or ranges of them. Requires: @ # & ~",
+			"- /wl reject [index1], [index2], ... [indexn] OR all - Remove questions from the Weakest Link submission database using their index numbers or ranges of them. Requires: @ # & ~",
+			"- /wl qs - View the questions in the Weakest Link database. Requires: % @ # & ~",
+		].join("<br >");
+		let display = target === 'all' ? `${games}<br /><br />${questions}` : target === 'questions' ? questions : games;
+		this.sendReplyBox(display);
+	},
+};
+const BANK_AMOUNTS = [1000, 2500, 5000, 10000, 25000, 50000, 75000, 125000];
+class WeakestLink extends Rooms.RoomGame {
+	constructor(room) {
+		super(room);
+		this.bank = 0;
+		this.amountToBank = 0;
+		if (room.gameNumber) {
```

Comment:
This should be in `Trivia`'s constructor, even though it doesn't use this at the moment